apps/checkout/.env

```
@@ -1,4 +1,4 @@
 REACT_APP_SALEOR_API_URL=$SALEOR_API_URL
 REACT_APP_CHECKOUT_APP_URL=$CHECKOUT_APP_URL
-PORT=3001
+PORT=5000
```

Please change to something else - it's used for AirPlay Receiver on macOS and it might be a little hard to find why that port is busy by the users (the process identified by ControlCentre for some reason)

```suggestion
PORT=3003
```